packages/fast-usdc/test/snapshots/fast-usdc.contract.test.ts.md

```diff
@@ -3048,7 +3048,7 @@ Generated by [AVA](https://avajs.dev).
           withdrawHandler: Object @Alleged: Liquidity Pool withdrawHandler {},
         },
         'Liquidity Pool_kindHandle': 'Alleged: kind',
-        NobleICA: 'Vow',
+        'NobleICA-connection-40': 'Vow',
```

Comment:
This is the originally requested one?  (new one should have connection-38)

Reply:
Yes; we have not regenerated fetched-chain-info